### PR TITLE
add striping to data table [SATURN-93]

### PR DESCRIPTION
**Pull request by panentheos**
This is a lightweight fix to improve readability of the data table when scanning across long rows.

### CODE REVIEW

File: src/components/DataTable.js [hunk 1]
```diff
@@ -179,7 +179,10 @@ export default ajaxCaller(class DataTable extends Component {
                       }
                     }
                   }, _.filter('visible', columnSettings))
-                ]
+                ],
+                styleCell: ({ rowIndex }) => {
+                  return rowIndex % 2 && { backgroundColor: colors.light(0.2) }
```
Comment (breilly2) [inline src/components/DataTable.js:184]: It's not exactly but it's _really_ close to the background color of the header row. Can we adjust one or the other to create a little more contrast?

Comment (panentheos) [reply src/components/DataTable.js:184]: I darkened the header slightly, how does that look?

Comment (breilly2) [reply src/components/DataTable.js:184]: Ever so slightly better, I guess. I know there isn't much room to work with here. FWIW, I like it better on the MacBook screen than on my external monitor.

TOL: I suspect all of this subtlety may be lost when projected.

**Review by breilly2 — APPROVED** [review]
Code looks good. Visually looks fine, and is an improvement no matter how subtle.

**Pull request merged**